app/models/users/setting.rb

```diff
@@ -0,0 +1,41 @@
+module Users
+  class Setting < ApplicationRecord
+    self.table_name_prefix = "users_"
+
+    belongs_to :user
```

Comment:
Added this line to establish the association from User::Setting back to User

Reply:
Do we need this side of the association? I.e. do we ever go from the settings back to the user object? Because if not I don't think there's much point in adding this. At least personally I usually only add the associations for the direction(s) I actually need.

Reply:
@citizen428 I encouraged Arit to add the association when I was talking to her. We were having some issues in tests but in general I don't think it hurts to tell Rails that a Setting belongs to a User since they have a 1:1 relationship. Even Rails's documentation somewhat hints at this https://guides.rubyonrails.org/v6.0/association_basics.html#choosing-between-belongs-to-and-has-one

Reply:
All good. As I said, this is a personal preference, I tend to only add associations I'm likely to use. If I'm only going to use it from one "end", I'll generally only add it there. But `ActiveRecord` is by far my least favorite thing in Rails, so I may be biased.